### PR TITLE
Extend offline page to include app section

### PR DESCRIPTION
**Pull request by ganicke**
Extend the pxt offline base page to include a section for the offline app.

Move this section to the top above **Web application**?

RE: #1852

### CODE REVIEW

File: docs/offline.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# @extends
+
+## Offline app #target-app
+
+The MakeCode editor is available as app which you can install on a computer with Windows or Mac OS. Once installed, the **[MakeCode Offline App](offline-app)** lets you create, run, and download your projects to the @boardname@. It works the same as the Web application does in your browser but it's a stand-alone application that will work when a connection to the internet is restricted or not available.
```
Comment (pelikhan) [inline docs/offline.md]: /offline-app?

**Comment by pelikhan** [comment]
Yes we should move the offline app on top of the page and use a full url in the link instead of just a hyperlink imo. The offline page reads a bit funny right now.

File: docs/offline.md (hunk 1 above)
Comment (ganicke) [reply docs/offline.md]: Yea, oops.

**Comment by ganicke** [comment]
Right, this was originally done for the offline browser use.

**Comment by pelikhan** [comment]
Right it evolved :)

**Comment by abchatra** [comment]
@ganicke Can we do the same change for https://makecode.mindstorms.com/offline as well?

**Comment by ganicke** [comment]
I think so. I take a look that one too.

**Pull request merged**